apache/nuttx
# stm32f4/stm32f401rc-rs485: add support to  userleds

## Summary
Add support to userleds on stm32f401rc-rs485 board.

## Impact
Users will be able to use userleds 

## Testing
- Enable LED driver and LED application on menuconfig.
- Build and flash the firmware
- Run "leds" and see the LEDS counting from 0000 to 1111.
